core/ledger/kvledger/state_listener_test.go

```diff
@@ -62,7 +62,7 @@ func TestStateListener(t *testing.T) {
 	require.NoError(t, sim2.SetState(namespace, "key3", []byte("value3")))
 	sim2.Done()
 
-	// Simulate tx3 - this neighter conflicts with tx1 nor with tx2
+	// Simulate tx3 - this nighter conflicts with tx1 nor with tx2
```

Comment:
@goodfirm It seems like 'neither' is the correct word, not 'nighter'.
```suggestion
	// Simulate tx3 - this neither conflicts with tx1 nor with tx2
```

Reply:
Thanks! Modified

Please review again